source/nodejs/adaptivecards-site/themes/adaptivecards/source/js/script.js

```diff
@@ -595,6 +595,20 @@ $(function () {
 		// Kick off one resize to fix all videos on page load
 	}).resize();
 
+    function makeTabIndicesZero() {
```

Comment:
Perhaps we should only make the indices zero if they're not set to a negative number? This way, if anything was explicitly removed from tab order, we don't change those.

Reply:
that's a good point -- fix inbound shortly